addon/components/polaris-dropzone/file-upload.js

```diff
@@ -0,0 +1,52 @@
+import Component from '@ember/component';
+import layout from '../../templates/components/polaris-dropzone/file-upload';
+import { computed } from '@ember/object';
+import { readOnly } from '@ember/object/computed';
+import { capitalize } from '@ember/string';
+
+const iconDragDrop = 'drag-drop';
+const assetFileUpload = '/ember-polaris/images/file-upload.svg';
+const assetImageUpload = '/ember-polaris/images/image-upload.svg';
```

Comment:
Have you tested that these paths work on a fresh app rather than one of our existing host apps?